src/Identity/Extensions.Core/src/UserManagerMetrics.cs

```diff
@@ -32,12 +32,12 @@ internal sealed class UserManagerMetrics : IDisposable
     public UserManagerMetrics(IMeterFactory meterFactory)
     {
         _meter = meterFactory.Create(MeterName);
-        _createCounter = _meter.CreateCounter<long>(CreateCounterName, "count", "The number of users created.");
-        _updateCounter = _meter.CreateCounter<long>(UpdateCounterName, "count", "The number of user updates.");
-        _deleteCounter = _meter.CreateCounter<long>(DeleteCounterName, "count", "The number of users deleted.");
-        _checkPasswordCounter = _meter.CreateCounter<long>(CheckPasswordCounterName, "count", "The number of check password attempts. Only checks whether the password is valid and not whether the user account is in a state that can log in.");
-        _verifyTokenCounter = _meter.CreateCounter<long>(VerifyTokenCounterName, "count", "The number of token verification attempts.");
-        _generateTokenCounter = _meter.CreateCounter<long>(GenerateTokenCounterName, "count", "The number of token generation attempts.");
+        _createCounter = _meter.CreateCounter<long>(CreateCounterName, "{user}", "The number of users created.");
+        _updateCounter = _meter.CreateCounter<long>(UpdateCounterName, "{user}", "The number of users updated.");
+        _deleteCounter = _meter.CreateCounter<long>(DeleteCounterName, "{user}", "The number of users deleted.");
```

Comment:
Is the unit supposed to be singular or plural? I see "{bytes}" in some places like PinnedBlockMemoryPoolMetrics, but then we use "{request}" in AuthorizationMetrics, and "{exception}" in DiagnosticsMetrics? I'm guessing it's PinnedBlockMemoryPoolMetrics that should change to be singular, but I'm not sure.

Reply:
it should always be singular

Reply:
Already a PR to update memory pool metrics here: https://github.com/dotnet/aspnetcore/pull/62766